week1-final(amin)/app/src/main/java/com/kodeco/android/coordplot/MainActivity.kt

```diff
@@ -0,0 +1,121 @@
+package com.kodeco.android.coordplot
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.offset
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.Slider
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.kodeco.android.coordplot.ui.theme.MyApplicationTheme
+
+class MainActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            MyApplicationTheme {
+                PlotSurface()
+            }
+        }
+    }
+}
+
+@Composable
+fun PlotSurface() {
+    Surface(
+        modifier = Modifier.fillMaxSize(),
+        color = Color.White
+    ) {
+        var xValue by remember { mutableStateOf(0.5f) }
+        var yValue by remember { mutableStateOf(0.5f) }
+
+        Column (
+            verticalArrangement = Arrangement.SpaceEvenly,
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+            Text(" Coordinate Plotter ",
+                fontSize = (36.sp)
+
+            )
```

Comment:
minor, but the formatting of the params in this `Text` composable can be updated so each param field is on it's own line.  This is how the [Kotlin coding conventions](https://kotlinlang.org/docs/coding-conventions.html#functions) recommends this approach as well:

```suggestion
            Text(
                text = " Coordinate Plotter ",
                fontSize = (36.sp),
            )
```

Note: You could also add the trailing comma after `fontSize` that way if you add any future params to the function call then you don't have to incur a line change in git by adding the comma later.